org/mozilla/jss/netscape/security/x509/AlgorithmId.java

```diff
@@ -566,7 +784,8 @@ public final boolean equals(ObjectIdentifier id) {
     private static final int SHA256_data[] = { 2, 16, 840, 1, 101, 3, 4, 2, 1 };
     private static final int SHA384_data[] = { 2, 16, 840, 1, 101, 3, 4, 2, 2 };
     private static final int SHA512_data[] = { 2, 16, 840, 1, 101, 3, 4, 2, 3 };
-
+    private static final int SHA224_data[] = { 2, 16, 840, 1, 101, 3, 4, 2, 4 };
```

Comment:
Just askin', but why was SHA224 added here? We don't really support SHA224 anywhere else in JSS.

Reply:
I was doing this way back when for completeness. I think I will just get rid of it.

Reply:
Got rid of it.